### PR TITLE
Fix: refresh on chunk load error due to deployment

### PR DESCRIPTION
**Pull request by riodeuno**

### CODE REVIEW

**Comment by riodeuno** [comment]
Converting this to draft due to false positives in the regex match for JS chunk load fails.

File: app/client/public/index.html [hunk 1]
```diff
@@ -41,10 +41,46 @@
...
+          }
+      }
+    };
     window.addEventListener("DOMContentLoaded", (event) => {
       document.getElementById("loader").style.width = "50vw";
     });
     window.addEventListener("load", (event) => {
+      // This signifies that we've had a chunk load failure in this session
+      // We clear the entry, as otherwise, we will have a repeated reload scenario.
+      window.localStorage.removeItem("donotReloadOnChunkLoadFail");
```
Comment (Nikhil-Nandagopal) [inline app/client/public/index.html:82]: @riodeuno will this trigger correctly? I thought this should happen in the above else clause

Comment (Nikhil-Nandagopal) [reply app/client/public/index.html:82]: This looks like it will clear as soon as we reload which will cause the infinite - reload

Comment (riodeuno) [reply app/client/public/index.html:82]: The handler is for the `load` event, which runs after all assets have loaded. So, the `onerror` event will be triggered before the `load` event.

**Review by mohanarpit — APPROVED** [review]

**Pull request merged**